nextcord/client.py

```diff
@@ -1467,6 +1468,34 @@ async def fetch_guild(self, guild_id: int, /, *, with_counts: bool = True) -> Gu
         data = await self.http.get_guild(guild_id, with_counts=with_counts)
         return Guild(data=data, state=self._connection)
 
+    async def fetch_guild_preview(self, guild_id: int, /) -> GuildPreview:
+        """|coro|
+
+        Fetches a :class:`.GuildPreview` from an ID.
+
+        .. note::
+          This will only fetch guilds that the bot is in or that are discoverable.
```

Comment:
```suggestion
        .. note::
        
          This will only fetch guilds that the bot is in or that are discoverable.
```

I *think* there needs to be a line between

Reply:
It still displays normally without the new line though